packages/opentelemetry/test/utils/parseSpanDescription.test.ts

```diff
@@ -172,6 +296,26 @@ describe('descriptionForHttpMethod', () => {
         source: 'url',
       },
     ],
+    [
+      'works with prefetch request',
```

Comment:
This is just an additional test I added because we didn't test the condition to add `.prefetch` to the `op`.